haskell-src/Concordium/Types/Queries.hs

```diff
@@ -732,8 +734,14 @@ $(deriveJSON defaultOptions{fieldLabelModifier = firstLower . drop (length ("_nu
 
 -- | Build the struct containing all of the next available sequence numbers for updating any of the
 -- chain parameters
-updateQueuesNextSequenceNumbers :: UQ.PendingUpdates cpv -> NextUpdateSequenceNumbers
-updateQueuesNextSequenceNumbers UQ.PendingUpdates{..} =
+updateQueuesNextSequenceNumbers ::
+    -- | The queues with pending updates
+    UQ.PendingUpdates cpv ->
+    -- | The update sequence number for protocol level tokens.
+    -- No queue is associated with this chain update, so it is provided separately.
+    U.UpdateSequenceNumber ->
+    NextUpdateSequenceNumbers
+updateQueuesNextSequenceNumbers UQ.PendingUpdates{..} protocolLevelTokensSeqNumber =
```

Comment:
It might be nicer to pass in the `updates` as input parameter to the function and then extract the `protocolLevelTokensSeqNumber` inside this function. Just thinking that this would be a better approach to handle more chain parameters in the future that do not have a queue.

Reply:
Yes, I agree. I did not do this because the `Updates` type lives in the `concordium-node` repo and did not want to move it into `concordium-base`.
Maybe this function could be moved to `concordium-node`

Reply:
I tried [searching across all our repos](https://github.com/search?q=org%3AConcordium%20updateQueuesNextSequenceNumbers&type=code) and it is only used in one place in concordium-node, so I have made it inline in the node instead.